editor.planx.uk/src/@planx/components/shared/PrivateFileUpload/Dropzone.tsx

```diff
@@ -80,6 +85,8 @@ export const Dropzone: React.FC<Props> = ({
 }) => {
   const MAX_UPLOAD_SIZE_MB = 30;
 
+  const isWithinListCard = Boolean(useOptionalListContext());
```

Comment:
This feels like a bit of a hacky solution to work out if the component is within a List or a Page but it works! 

A good alternate solution here would be CSS container queries but these aren't supported in our version of MUI.